package.json

```diff
@@ -4,6 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start": "babel-node index.js",
```

Comment:
You shouldn't use `babel-node` on production. You'll need to add a build script that will always build the app on every deploy. The app should be started using node (`node build/index.js`)

Reply:
done.